test/e2e/instrumentation/logging/stackdrvier/utils.go

```diff
@@ -49,6 +49,9 @@ const (
 
 	// The parallelism level of polling logs process.
 	sdLoggingPollParallelism = 10
+
+	// The limit on the number of stackdriver sinks that can be created within one project.
```

Comment:
I'd mention here that this code will remove all sinks after exceeding this limit.

Reply:
I don't think we should connect const to description of how it is used. Maybe in future we will clean sinks when we hit 90% of limit instead of 100%. 

As for commenting function I thinks it's simple enough. I would prefer to rename/refactor it instead of adding comments